napari/layers/image/_slice.py

```diff
@@ -147,21 +221,28 @@ def _call_multi_scale(self) -> _ImageSliceResponse:
             ndim=self.dims.ndim,
         )
 
-        thumbnail_indices = self._slice_indices_at_level(self.thumbnail_level)
-
-        image = self.data[level][tuple(indices)]
-        thumbnail = self.data[self.thumbnail_level][tuple(thumbnail_indices)]
+        order = self._get_order()
+        data = np.asarray(self.data[level][tuple(indices)])
+        data = np.transpose(data, order)
+        image = _ImageView.from_view(data)
 
-        if not self.lazy:
-            image = np.asarray(image)
-            thumbnail = np.asarray(thumbnail)
+        thumbnail = image
```

Comment:
NIT: consider moving to an `else` below so we dont create and then sometimes overwrite